aws/logs_monitoring/README.md

```diff
@@ -75,6 +75,12 @@ You have two options to add custom tags to your logs:
 
 ## 3. (optional) Send logs to EU or to a proxy
 
+### Send logs through TCP
+
+By default, the forwarder sends logs using HTTPS through the port 443. 
+
+If you would rather use raw TCP, set the environment variable `DD_USE_TCP` to `true`.
```

Comment:
I would replace this by: `To send logs over a SSL encrypted TCP connection, set ....`